### PR TITLE
Add Xorima to CodeOwners

### PR DESCRIPTION
**Pull request by xorima**
### Description

Adds Xorima to be a code owner

### Issues Resolved

N/A

### Check List
- [ ] All tests pass. See https://github.com/sous-chefs/grafana/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please add labels to this Pull Request</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @damacus @scopenco 
+* @damacus @scopenco @xorima
```
Comment (damacus) [inline CODEOWNERS]: `@sous-chefs/grafana ` 

I think we might want this instead 🚀

Comment (xorima) [reply CODEOWNERS]: I was just thinking the same, give me a moment :)

File: CODEOWNERS [hunk 2]
```diff
@@ -1 +1 @@
-* @damacus @scopenco 
+* @damacus @scopenco @sous-chefs/grafana
```
Comment (damacus) [inline CODEOWNERS]: You can remove `@damacus` and `@scopenco`

**Review by damacus — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 2 above)
Comment (xorima) [reply CODEOWNERS]: Done, but neither of you are in the group fyi

**Review by damacus — APPROVED** [review]

**Pull request merged**